src/static/signup.php

```diff
@@ -7,4 +7,7 @@
 
 $connect = mysql_connect($dbservername, $dbusername, $dbpassword); 
```

Comment:
this is another page that can be deleted. register.php has taken the place of this.